README.md

```diff
@@ -1,70 +1,180 @@
+# 📖 Hello Microverse <a name="about-project"></a>
 
-### `npm run build`
+**CoinStats** is a React/Redux SPA that fetches and renders data from the [CoinGecko API](https://www.coingecko.com/en/api).
+It allows users to see market data for each of the top 100 coins in the crypto Market and Features a Convertion Filter that alows you to see te value of each coin in different currencies, as well as converting different amounts (filter works in a two way fashion).
+To easily find a coin you are interested, use the search bar filter in the main page.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## 🛠 Built With <a name="built-with"></a>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Tech Stack <a name="tech-stack"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/html/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">SCSS</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
 
-### `npm run eject`
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://render.com/">render</a></li>
+  </ul>
+</details>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+<!-- Features -->
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Key Features <a name="key-features"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- Market data for:
+  - **Global Market Cap**
+  - **24hr Volume**
+  - **Top 100 cryptocurrencies**
+- Main Market data for each of the Top 100 criptocurrencies
+- **Price convertion filter** to multiple currencies for each of the Top 100 coins. **Filter handles different amounts in both sides**.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<!-- LIVE DEMO -->
 
-## Learn More
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [Live Demo Link](https://hello-microverse.netlify.app/) -->
```

Comment:
- [ ] Please kindly include the actual deployment for this project. At this moment, this deployment is for the `Hello Microverse` project.